nss-cohort-36/bangazon-api-virtual_magicians
# Cb getting started

# Description
Added data to Products to filter by location

## Type of change
Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout cb-getting-started`
4. `python manage.py loaddata producttypes`
5. `python manage.py loaddata products`
6. go to client side PR to continue

# Checklist:
